api/.env-template

```diff
@@ -44,3 +44,6 @@ AWS_PROFILE_PROD=
 AWS_PROFILE_DEV=
 DYNAMODB_TABLE_DEV=
 DYNAMODB_TABLE_PROD=
+TAX_CLEARANCE_CERTIFICATE_URL=
```

Comment:
remove since we are not using this locally

Reply:
This feels similar to the ABC stuff we were talking about in Slack. I really could see this either way. I just think it should be consistent.

Reply:
Decision after chatting with Sam: we are storing the environment variables in BitWarden and the `.env-template.`